### PR TITLE
Identify first time contributors

### PR DESCRIPTION
**Pull request by OmarShehata**
This implements the feature described here https://github.com/AnalyticalGraphicsInc/cesium-concierge/issues/85.

I branched off of https://github.com/AnalyticalGraphicsInc/cesium-concierge/pull/118 because I needed the contributors URL fix from there. This just checks the `CONTRIBUTORS.md` in the repo/branch you're merging _into_. If the user's name isn't there, they are declared to be a first time external contributor and concierge will @ mention outreach team members. I didn't bother differentiating between internal/external contributors because if it's someone's first contribution they are 99% probably an external contributor. 

To do when merged:

* [ ] Updated templates in repo's to enable the firstContribution message.

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @OmarShehata! 
* :heavy_check_mark: Signed CLA found. 
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium-concierge/blob/first-time/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

---

:sparkles: :sparkles: :sparkles:__I'm the Cesium Concierge and I'm here to test things.__ :sparkles: :sparkles: :sparkles:

File: lib/templates/issueClosed.hbs [hunk 1]
```diff
@@ -1,3 +1,9 @@
+{{#if isFirstContribution}}
+Thanks for the awesome work @{{ userName }}! Your contribution is about to launch to millions of users with the next release :rocket:
```
Comment (ggetz) [inline lib/templates/issueClosed.hbs]:
```suggestion
Thanks for the awesome work @{{ userName }}! Your contribution is about to launch to millions of users with the next release. :rocket:
```

File: README.md [hunk 2]
```diff
@@ -36,11 +36,12 @@ The table below describes all the possible configuration variables, as well as t
...
-| `repositories:{full_name}:contributorsUrl` | `string` |  The GitHub API URL to `CONTRIBUTORS.md`. | X | _Disabled if not set._
+| `repositories:{full_name}:contributorsPath` | `string` |  Relative path to the `CONTRIBUTORS.md` file. | X | _Disabled if not set._
 | `repositories:{full_name}:maxDaysSinceUpdate` | `number` | "Bump" pull requests older than this number of days ago. | X | `30`
 | `repositories:{full_name}:unitTestPath` | `string` |  Relative path to the directory containing unit tests. _Example:`Specs/`_ | X | _Disabled if not set._
-| `port` | `number` | Port on which to listen to incoming requests | X | `5000`
-| `listenPath` | `string` | Path on which to listen for incoming requests | X | `"/"`
+| `port` | `number` | Port on which to listen to incoming requests. | X | `5000`
+| `listenPath` | `string` | Path on which to listen for incoming requests. | X | `"/"`
+| `outreachUsers` | `string` | Username(s) to mention when someone makes their first contribution. Example:`@OmarShehata`. You can add multiple people or prepend it with a message this way. | X | _Disabled if not set._
```
Comment (ggetz) [inline README.md]: ` prepend it with a message this way` - a little vague, an example of this use case would help.

Comment (ggetz) [reply README.md]: Actually, do we need this value in the config file at all? Why not just specify it in the first time contribution message templates?

File: specs/lib/commentOnClosedIssueSpec.js [hunk 3]
```diff
@@ -190,12 +237,75 @@ describe('commentOnClosedIssue', function () {
...
+        commentOnClosedIssue._implementation(options)
             .then(done.fail)
             .catch(function (error) {
                 expect(error).toEqual('Bad request');
                 expect(requestPromise.post).not.toHaveBeenCalled();
                 done();
             });
     });
+
+    it('commentOnClosedIssue._implementation posts about first timer\'s contribution.', function (done) {
```
Comment (ggetz) [inline specs/lib/commentOnClosedIssueSpec.js]: Let's also test that the message don't get included when it's not their first contribution.

**Comment by OmarShehata** [comment]
Thanks for the super speed review @ggetz ! And you were right, it's a lot simpler just listing outreach users in the template itself as opposed to adding a config. I added a test to make sure it doesn't post if a user is not a first time contributor, fixed that missing period in the template, and added a TODO bullet point in this PR to remember to update the templates in the relevant repositories.

**Comment by ggetz** [comment]
This is good to go. Let's check https://github.com/AnalyticalGraphicsInc/cesium-concierge/pull/126 works before deploying this change as well. And, of course, update the relevant repos when you deploy as well!

**Comment by cesium-concierge** [comment]
Thanks again for your contribution @OmarShehata! 

No one has commented on this pull request in 30 days. Maintainers, can you review, merge or close to keep things tidy?

 _I'm going to re-bump this in 30 days. If you'd like me to stop, just comment with `@cesium-concierge stop`. If you want me to start again, just delete the comment._

---

:sparkles: :sparkles: :sparkles:__I'm the Cesium Concierge and I'm here to test things.__ :sparkles: :sparkles: :sparkles:

**Comment by OmarShehata** [comment]
Once deployed, this should start working right away, since the Cesium repo uses the default issueClosedTemplate which will tag me and Sarah. We can change that here or in the Cesium repo if we want to change who to tag.

**Pull request merged**